test/integration/integration-test.js

```diff
@@ -124,45 +144,201 @@ describe.skip('integration tests', function() {
       direction: 'buy',
       quantity: {
         currency: 'USD',
-        value: '100',
-        counterparty: 'r3PDtZSa5LiYp1Ysn1vMuMzB59RzV3W9QH'
+        value: '237',
+        counterparty: 'rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q'
       },
       totalPrice: {
         currency: 'XRP',
-        value: '0.000001'
-      },
-      immediateOrCancel: true
+        value: '0.0002'
+      }
     };
     const self = this;
     const lastClosedLedgerVersion = this.api.getLedgerVersion();
     async.waterfall([
-      this.api.prepareOrder.bind(this.api, wallet.getAddress(), orderSpecification, instructions),
+      this.api.prepareOrder.bind(this.api, wallet.getAddress(),
+        orderSpecification, instructions),
       _.partial(verifyResult, 'OfferCreate'),
       _.partial(testTransaction, this, 'order', lastClosedLedgerVersion),
       (txData, callback) => {
-        self.api.prepareOrderCancellation(wallet.getAddress(), txData.Sequence, instructions, callback);
+        this.api.getOrders(wallet.getAddress(), {}, composeAsync(orders => {
+          assert(orders && orders.length > 0);
+          const createdOrder = _.first(_.filter(orders, (order) => {
+            return order.properties.sequence === txData.Sequence;
+          }));
+          assert(createdOrder);
+          assert.strictEqual(createdOrder.properties.maker,
+            wallet.getAddress());
+          assert.deepEqual(createdOrder.specification, orderSpecification);
+          return txData;
+        }, callback));
+      },
+      (txData, callback) => {
+        self.api.prepareOrderCancellation(wallet.getAddress(), txData.Sequence,
+          instructions, callback);
       },
       _.partial(verifyResult, 'OfferCancel'),
-      _.partial(testTransaction, this, 'orderCancellation', lastClosedLedgerVersion)
+      _.partial(testTransaction, this, 'orderCancellation',
+        lastClosedLedgerVersion)
     ], () => done());
   });
 
+
+  it('isConnected', function() {
+    assert(this.api.isConnected());
+  });
+
+
+  it('getServerInfo', function(done) {
+    this.api.getServerInfo(composeAsync(data => {
+      assert(data && data.info && data.info.pubkey_node);
+    }, done));
+  });
+
+
+  it('getFee', function() {
+    const fee = this.api.getFee();
+    assert.strictEqual(typeof fee, 'string');
+    assert(!isNaN(+fee));
+  });
+
+
+  it('getLedgerVersion', function() {
+    const ledgerVersion = this.api.getLedgerVersion();
+    assert.strictEqual(typeof ledgerVersion, 'number');
+    assert(ledgerVersion > this.startLedgerVersion);
```

Comment:
Yeah, we should always assume that all the test cases are independent, so it should be possible to run it by itself, which is often done for debugging using `mocha -g`.